### ISSUE TITLE
mask_logits function

### ISSUE THREAD

**Issue by BangLiu** [andy840314/QANet-pytorch-#5]
I think in mask_logits, maybe "return inputs + (-1e30) * (1 - mask)" shall be replaced by "return inputs * mask + (-1e30) * (1 - mask)".

**Comment by andy840314** [comment]
I don't think it will make a big difference since -1e30 is already a extremely small value and should mask correctly.
